crates/api_models/src/admin.rs

```diff
@@ -392,7 +392,7 @@ pub struct MerchantConnectorCreate {
     pub connector_type: api_enums::ConnectorType,
     /// Name of the Connector
     #[schema(example = "stripe")]
```

Comment:
```suggestion
    #[schema(value_type = Connector, example = "stripe")]
```

Please re-generate the OpenAPI spec once you've made this change.